libs/wire-api/src/Wire/API/Team.hs

```diff
@@ -227,6 +229,10 @@ instance Arbitrary BindingNewTeam where
 -- | FUTUREWORK: this is dead code!  remove!
 newtype NonBindingNewTeam = NonBindingNewTeam (NewTeam (Range 1 127 [TeamMember]))
   deriving stock (Eq, Show, Generic)
+  deriving (S.ToSchema) via (Schema NonBindingNewTeam)
+
+instance ToSchema NonBindingNewTeam where
+  schema = error "todo"
```

Comment:
I missed this, will fix it

Reply:
argh!!  we should not have non-binding teams any more in the code, they have never been implemented on the client side and have been dead code since the beginning.  i expect this to take less than an hour, but now i want to remove non-binding teams again.  immediately!